Sources/Commands/SwiftBuildTool.swift

```diff
@@ -173,13 +173,13 @@ public struct SwiftBuildTool: SwiftTool {
         print("USAGE: swift build [mode] [options]")
         print("")
         print("MODES:")
-        print("  -c, --configuration <value>   Build with configuration (debug|release)")
-        print("  --clean <mode>                Delete artifacts (build|dist)")
+        print("  -c, --configuration <value>   Build with configuration (default: debug|release)")
+        print("  --clean <mode>                Delete artifacts (default: build|dist)")
         print("")
         print("OPTIONS:")
         print("  -C, --chdir <path>       Change working directory before any other operation")
-        print("  --build-path <path>      Specify build directory")
-        print("  --color <mode>           Specify color mode (auto|always|never)")
+        print("  --build-path <path>      Specify build directory, default: ./.build")
+        print("  --color <mode>           Specify color mode (default: auto|always|never)")
```

Comment:
This _might_ be confusing, How about:
`--color <mode>           Specify (auto|always|never) [default: auto]`